Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [InnerTune/Main strings](https://hosted.weblate.org/projects/innertune/app/).



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widget/innertune/app/horizontal-auto.svg)

wdyk28 committed:
Translated using Weblate (Hindi)

Currently translated at 68.2% (172 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/hi/

Matko802 committed:
Added translation using Weblate (Slovak)

lucasmz-dev commented:
@Matko802 Do you plan on translating to Slovak or not? I'll delete the empty language if not

Matko802 commented:
Wdym by empty language

Matko802 commented:
> @Matko802 Do you plan on translating to Slovak or not? I'll delete the empty language if not

Yes i will translate it im new to weblate but ill try

Matko802 commented:
> > Wait is innertune connected to metrolist?
> 
> Yeah, Metrolist is an InnerTune fork of course (even if the Git history was reset), but I also went and helped Metrolist centralize and deduplicate the translation work by contributors. Originally done also in OuterTune.
> You need to translate InnerTune for the man strings to land in Metrolist. This does mean your work will survive outside of just Metrolist.


Oh so i need to translate innertune too?

lucasmz-dev commented:
yes

Matko802 commented:
> yes

I will translate it everything

Mickael81 committed:
Translated using Weblate (French)

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/fr/

Matko802 committed:
Translated using Weblate (Slovak)

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/sk/

Matko802 commented:
> Thanks mat! It will appear soon in Metrolist.
> You might get some notifications here for when weblate updates things, you may want to unsubscribe

Also how does it work like do the app needs to be updated or weblate will automatically put it into app without updating anything?

lucasmz-dev commented:
> Also how does it work like do the app needs to be updated or weblate will automatically put it into app without updating anything?

Don't worry, this PR doesn't need to be merged into InnerTune itself for it to arrive in the forks. InnerTune's development is paused at the moment anyway. I manually send PRs to update them to both Metrolist and OuterTune. If InnerTune itself goes completely silent, I plan to keep this working.

Matko802 commented:
> Speaking of, your stuff just got merged into Metrolist.

how when its not there in app language settings

lucasmz-dev commented:
Things being merged doesn't mean the version you have contains it. It will be there next version.

Matko802 commented:
> Things being merged doesn't mean the version you have contains it. It will be there next version.

oh now i understand

lucasmz-dev commented:
Metrolist hasn't had in-app language settings for a while now, as they were very broken, but you would be able to select it in Android 13+ devices under app info. Are you sure you're also not very out of date? I recommend getting F-Droid and adding the IzzyOnDroid repo and update using it.

Matko802 commented:
> Metrolist hasn't had in-app language settings for a while now, as they were very broken, but you would be able to select it in Android 13+ devices under app info. Are you sure you're also not very out of date? I recommend getting F-Droid and adding the IzzyOnDroid repo and update using it.

also from izzydroid directly the like button is bugged when i update from github it fixes itself

mardous committed:
Translated using Weblate (Spanish (American))

Currently translated at 67.0% (169 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/es_US/

Commit:
Translated using Weblate (Spanish (American))

Currently translated at 67.0% (169 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/es_US/

milotype committed:
Translated using Weblate (Croatian)

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/hr/

Valer100 committed:
Translated using Weblate (Romanian)

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/ro/

asdhgiuasdh80u committed:
Added translation using Weblate (Filipino)

adham-debug committed:
Added translation using Weblate (Azerbaijani)

Commit:
Translated using Weblate (Punjabi)

Currently translated at 78.1% (197 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/pa/

Commit:
Added translation using Weblate (Swedish)

jvpag committed:
Added translation using Weblate (Basque)

jvpag committed:
Translated using Weblate (Basque)

Currently translated at 0.3% (1 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/eu/

Commit:
Translated using Weblate (Spanish (American))

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/es_US/

lucasmz-dev committed:
Translated using Weblate (English (Canada))

Currently translated at 0.7% (2 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/en_CA/

Matko802 committed:
Translated using Weblate (Slovak)

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/sk/

Matko802 commented:
@lucasmz-dev I made small mistake instead of home i put there bottom navigation in slovak language but i fixed it now. 😅

milotype committed:
Translated using Weblate (Croatian)

Currently translated at 100.0% (3 of 3 strings)

Translation: InnerTune/App store details
Translate-URL: https://hosted.weblate.org/projects/innertune/fastlane/hr/

milotype committed:
Translated using Weblate (Croatian)

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/hr/

weblate committed:
Update translation files

Updated by "Remove blank strings" add-on in Weblate.

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/

milotype committed:
Translated using Weblate (Croatian)

Currently translated at 100.0% (3 of 3 strings)

Translation: InnerTune/App store details
Translate-URL: https://hosted.weblate.org/projects/innertune/fastlane/hr/

milotype committed:
Translated using Weblate (Croatian)

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/hr/

arya12hama1 committed:
Added translation using Weblate (Kurdish (Central))

lucasmz-dev commented:
@arya12hama1 please translate if you create a language

cyberboh committed:
Translated using Weblate (Indonesian)

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/id/

lucasmz-dev committed:
Deleted translation using Weblate (Basque)

lucasmz-dev committed:
Deleted translation using Weblate (Azerbaijani)

lucasmz-dev committed:
Deleted translation using Weblate (Filipino)

lucasmz-dev committed:
Deleted translation using Weblate (Kurdish (Central))

lucasmz-dev committed:
Deleted translation using Weblate (Swedish)

JIRINKO committed:
Added translation using Weblate (Thai)

Drsheppard01 committed:
Translated using Weblate (Russian)

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/ru/

lucasmz-dev committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (252 of 252 strings)

Translation: InnerTune/Main strings
Translate-URL: https://hosted.weblate.org/projects/innertune/app/pt_BR/

TheBrch committed:
Added translation using Weblate (Lithuanian)

lucasmz-dev committed:
Added translation using Weblate (Portuguese (Portugal))

lucasmz-dev committed:
Deleted translation using Weblate (Portuguese (Portugal))

Fr33z3Fir3 committed:
Added translation using Weblate (Filipino)
